demos/python_demos/image_embeddings/servable/graph_pipeline.pbtxt

```diff
@@ -0,0 +1,72 @@
+input_stream: "OVMS_PY_TENSOR: image"
+output_stream: "OVMS_PY_TENSOR: embedding"
+
+node{
+    name: "ImagePreprocessor"
+    calculator: "PythonExecutorCalculator"
+    input_stream: "OVMS_PY_TENSOR: image"
+    output_stream: "OVMS_PY_TENSOR: pixel_values"
+    node_options:{
+        [type.googleapis.com/mediapipe.PythonExecutorCalculatorOptions]:{
+            handler_path=/workspace/pre.py
```

Comment:
are you sure this works? did the mediapipe servable become available? I think this is syntax error here, you should use `:` instead of `=`